tests/sentry/integrations/utils/test_code_mapping.py

```diff
@@ -106,16 +106,16 @@ def test_buckets_logic():
 
 
 class TestFrameFilename:
-    def test_frame_filename_package_and_more_than_one_level(self):
-        ff = FrameFilename("getsentry/billing/tax/manager.py")
-        assert f"{ff.root}/{ff.dir_path}/{ff.file_name}" == "getsentry/billing/tax/manager.py"
-        assert f"{ff.dir_path}/{ff.file_name}" == ff.file_and_dir_path
+    # def test_frame_filename_package_and_more_than_one_level(self):
```

Comment:
It's better to use `pytest.mark.skip("reason for skipping")`
If you're timing to land this tomorrow please go ahead and fix it in this PR.

Reply:
Added the skip, but the code still needs to be commented out to avoid mypy complaining and sabotaging the CI